specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+            "provisioningState": {
+              "readOnly": true,
+              "description": "(Read only.) The status of the Cosmos DB account at the time the operation was called. The status can be one of following. 'Creating' – the Cosmos DB account is being created. When an account is in Creating state, only properties that are specified as input for the Create Cosmos DB account operation are returned. 'Succeeded' – the Cosmos DB account is active for use. 'Updating' – the Cosmos DB account is being updated. 'Deleting' – the Cosmos DB account is being deleted. 'Failed' – the Cosmos DB account failed creation. 'DeletionFailed' – the Cosmos DB account deletion failed.",
+              "type": "string"
+            },
+            "restoreFromBackup": {
+              "type": "string",
+              "description": "Omit or set to null to create a new, empty cluster. If you are creating  new cluster by restoring a backup, then provide the unique resource identifier of the backup."
+            },
+            "delegatedManagementSubnetId": {
+              "type": "string",
+              "description": "(Can be set only at creation time.) Resource ID of a subnet that this cluster's management service should run in."
+            },
+            "cassandraVersion": {
+              "type": "string",
+              "description": "Which version of Cassandra should this cluster converge to running (e.g., 3.11). When updated, the cluster may take some time to migrate to the new version."
+            },
+            "overrideClusterName": {
+              "type": "string",
+              "description": "(Can be set only at creation time.) Optional value to use for clusterName in cassandra.yaml. If omitted, use the cluster's resource name."
+            },
+            "authenticationMethod": {
+              "type": "string",
+              "description": "Which authentication method Cassandra should use to authenticate clients.",
+              "enum": [
+                "None",
+                "Cassandra"
+              ]
+            },
+            "initialCassandraAdminPassword": {
+              "description": "(Can be set only at creation time.) Initial password for clients connecting as admin to the cluster. Should be changed after cluster creation. Returns null on GET. This field only applies when the authenticationMethod field is \"Cassandra\".",
+              "type": "string"
+            },
+            "hoursBetweenBackups": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Number of hours between backups, or 0 for no backups."
+            },
+            "repairEnabled": {
+              "type": "boolean",
+              "description": "Should automatic repairs run on this cluster? If omitted, this is true, and should stay true unless you are running a hybrid cluster where you are already doing your own repairs."
+            },
+            "clientCertificates": {
```

Comment:
consider grouping related properties into one object eg - the 3 cert related ones, 2 seedNode related ones, and 2 auth related ones

Reply:
I had that, and the linters complained about it. Which do I believe?